package.json

```diff
@@ -19,7 +20,8 @@
     "release:firefox": "web-ext build --source-dir dist/src --overwrite-dest --artifacts-dir dist/",
     "release": "npm run release:chrome && npm run release:firefox",
     "presign": "npm run release",
-    "sign": "web-ext sign --source-dir dist/src --artifacts-dir dist/ --id \"{d403ee9c-3bd2-41d3-b1e9-27698babf097}\""
+    "sign": "web-ext sign --source-dir dist/src --artifacts-dir dist/ --id \"{d403ee9c-3bd2-41d3-b1e9-27698babf097}\"",
+    "watch": "rollup -c -w"
```

Comment:
I'd add a dependency to `build:static` like `npm run build:static && rollup -c -w` so people don't forget to build css and manifest file

Reply:
i add concurrently package to run both `build:static` and `rollup -c -w` simultaneously